_CodingChallenges/153-interactive-sketchrnn.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Interactive Drawing with SketchRNN"
+video_number: 153
+date: 2019-10-24
+video_id: ZCXkvwLxBrA
+web_editor: hcumr-aua
+repository: CC_153_Interactive_SketchRRN
+
+links:
+  - title: "ml5.js"
+    url: "https://ml5js.org/"
+  - title: "Draw Together with a Neural Network - Magenta TensorFlow"
+    url: "https://magenta.tensorflow.org/assets/sketch_rnn_demo/index.html"
+  - title: "Ramer-Douglas-Peucker Algorithm on Wikipedia"
+    url: https://en.wikipedia.org/wiki/Ramer%E2%80%93Douglas%E2%80%93Peucker_algorithm
+  - title: "A Neural Representation of Sketch Drawings"
+    url: "https://arxiv.org/abs/1704.03477"
+  - title: "Quick, Draw!"
+    url: "https://quickdraw.withgoogle.com/#"
+  - title: "Recurrent Neural Networks & LSTMs"
+    url: "https://ayearofai.com/rohan-lenny-3-recurrent-neural-networks-10300100899b"
+
+videos:
+  - title: "Challenge #128: Sketch-RNN Snowflakes"
+    author: "The Coding Train"
+    url: "/CodingChallenges/128-sketchrnn-snowflakes"
+  - title: "Challenge #152: RDP Algorithm"
+    author: "The Coding Train"
+    url: "/CodingChallenges/152-rdp-algorithm"
+---
+
+This challenge uses SketchRNN from ml5.js to create a program where the person starts the drawing and computer will finish it using whatever SketchRNN model is loaded.
```

Comment:
```suggestion
This challenge uses  the pre-trained SketchRNN machine learning model (available with the ml5.js library) to create a p5.js sketch that finishes a person's drawing.
```

Minor changes, thank you for the description! I try not to be too wordy, but I added some stuff here which is often good for YouTube search keywords.